src/core/index.ts

```diff
@@ -514,6 +515,22 @@ export class ThirdwebSDK implements IThirdwebSdk {
   }
 
   public async createSnapshot(leafs: string[]): Promise<SnapshotInfo> {
+    const counts: { [key: string]: any } = {};
```

Comment:
we should change this to if there's non valid address, we should throw an error. so it's more similar to current behavior where when there's duplicate there'll be error.

Reply:
hmm already checks for both invalid address and repeated addresses, tho added customer errors instead of `invariants` rn